packages/build/package.json

```diff
@@ -49,13 +50,13 @@
     "@types/tar-fs": "^2.0.0",
     "@types/tmp": "^0.2.3",
     "cross-spawn": "^7.0.3",
-    "depcheck": "^1.4.3",
+    "eslint": "^7.25.0",
     "sinon-chai": "^3.5.0",
-    "yaml": "^2.3.1"
```

Comment:
Same as the .depcheckrc comment above, I don't know why this had to be removed here in my PR.

Reply:
What was failing here? Maybe I just applied the wrong verion of `yaml` here? (Could be that it should have been `"yaml": "^1.10.0"` like in the top-level package.json?)